### ISSUE TITLE
w2v_embeddings_colors.p file missing

### ISSUE THREAD

**Issue by mingo-x** [superhans/colorfromlanguage#1]
Interesting work!

When I was trying to run the model, I found the w2v_embeddings_colors.p file missing from the priors folder. I wonder if this is a word2vec pre-trained embedding, or did you train the embedding yourself?

**Comment by superhans** [comment]
Oops. I forgot to add that into the repo. Added now.

**Comment by mingo-x** [comment]
Thank you!

Btw, in the paper, you used the model structure from "Colorful Image Colorization". But I noticed that in this repo, you switched to a res-net style structure. Is there any reason behind this?

**Comment by superhans** [comment]
Yes. I will release the code for the models in the paper shortly (film and concat), and in fact, they perform slightly better because it's weights prior was initialized from Zhang et al, "Colorful Image Colorization", ECCV 2016.
